README.md

```diff
@@ -105,9 +108,9 @@ export default defineConfig({
 
 ### Parameters
 
-- **moduleId**: Name of the virtual module, default @api.
-- **server**: The main file to build as the server app. [See default file.](./example/.api/server.js)
-- **handler**: The main file to register the API. It is called in viteServer and is the default entry. [See default file.](./example/.api/handler.js)
```

Comment:
the .api/handler.js (see cacheDir = '.api') is a temp file, when you build the project, please include this file in the version

Reply:
Yes i know its a temp file, but it is in the `node_modules` (at least by default), thats why it has never been in the build before. I can change the cacheDir to be outside of the `node_modules` if you want.

Reply:
Actually I just noticed that in the `.gitignore` file the `.api` folder is marked as ignored. Therefore, even when building with the `.api` folder being outside of the `node_modules`, it would not be commited.

@yracnet do you want me to change that?